Volume/Views/Onboarding/PublicationDetail/PublicationDetail.swift

```diff
@@ -112,16 +112,14 @@ struct PublicationDetail: View {
         Section {
             ScrollView {
                 backgroundImage
-
                 PublicationDetailHeader(navigationSource: navigationSource, publication: publication)
                     .padding(.bottom)
-
                 Divider()
                     .background(Color.volume.buttonGray)
                     .frame(width: 100)
-
                 Header("Articles")
                     .padding()
+                
```

Comment:
I have absolutely no clue what the new line conventions in this project are

Reply:
Generally `if` and `switch` blocks need newlines around them, and UI components should have no newlines surrounding them unless it's a `Spacer`. That's what I've noticed and been following that.